src/mono/wasm/Wasm.Build.Tests/Wasm.Build.Tests.csproj

```
@@ -44,7 +44,7 @@
     <ProjectReference Include="$(RepoRoot)src\tasks\Microsoft.NET.Sdk.WebAssembly.Pack.Tasks\Microsoft.NET.Sdk.WebAssembly.Pack.Tasks.csproj" />
     <Compile Include="$(BrowserProjectRoot)debugger\DebuggerTestSuite\BrowserLocator.cs" />
 
-    <None Include="$(RepoRoot)\src\mono\wasm\test-main.js" CopyToOutputDirectory="PreserveNewest" />
+    <None Include="$(RepoRoot)\src\mono\browser\test-main.js" CopyToOutputDirectory="PreserveNewest" />
```

Can we use `BrowserProjectRoot`?

```suggestion
    <None Include="$(BrowserProjectRoot)\test-main.js" CopyToOutputDirectory="PreserveNewest" />
```